infrastructure/templates/public-api/components/containerApp.bicep

```diff
@@ -137,7 +137,7 @@ resource containerApp 'Microsoft.App/containerApps@2023-11-02-preview' = {
         external: true
         targetPort: containerAppTargetPort
         allowInsecure: false
-        corsPolicy: corsPolicy
+        corsPolicy: any(corsPolicy)
```

Comment:
Haha nice one - I'd previously tried to fix this and was rejected by the deploy process for my changes, so this is a good trick!

Reply:
Not ideal though! I think the type we've used is actually correct, but if you try and set empty arrays as defaults, ARM will error out saying that it's invalid! The type checker seems to be lying to us by suggesting the properties aren't optional.

Reply:
Yeah I remember!